main.js

```diff
@@ -186,28 +217,149 @@ function toggleSelectElement(event) {
     })
 }
 
-/* Helpers */
+/*
+****************************************************** Layout ******************************************************
+*/
 
-function isTitleVisible() {
-    return titleVisibleToggle.textContent === "Set Hidden"
+function createLayout(items, gridWidth, gridHeight) {
+    // The layout data object. Muuri will read this data and position the items
+    // based on it.
+    var layout = {
```

Comment:
One var got left it :)

Reply:
Those vars are sneaky, changed to const.